src/rurl/url.c

```diff
@@ -6,6 +6,10 @@
 #include <stdio.h>
 #include <string.h>
 
+#ifndef RCHEEVOS_URL_PROTOCOL
+#define RCHEEVOS_URL_PROTOCOL "http"
+#endif
```

Comment:
If we want to go the `RCHEEVOS_SSL_HOST` route, would it look like this?
``` suggestion
#ifndef RCHEEVOS_URL_PROTOCOL
#ifdef RCHEEVOS_SSL_HOST
#define RCHEEVOS_URL_PROTOCOL "https"
#else
#define RCHEEVOS_URL_PROTOCOL "http"
#endif
#endif
```

Reply:
I'd have left the "#ifndef RCHEEVOS_URL_PROTOCOL" out. it's not like there's any other reasonable choices.
```
#ifdef RCHEEVOS_SSL_HOST
#define RCHEEVOS_URL_PROTOCOL "https"
#else
#define RCHEEVOS_URL_PROTOCOL "http"
#endif
```

Reply:
Sounds good.